cmd/mayactl/app/command/volumes_list.go

```diff
@@ -59,19 +59,19 @@ func (c *CmdVolumeOptions) RunVolumesList(cmd *cobra.Command) error {
 	var cvols v1alpha1.CASVolumeList
 	err := mapiserver.ListVolumes(&cvols)
 	if err != nil {
-		return fmt.Errorf("Volume list error: %s", err)
+		return fmt.Errorf("volume list error: %s", err)
 	}
 
 	out := make([]string, len(cvols.Items)+2)
-	out[0] = "Namespace|Name|Status|Type"
-	out[1] = "---------|----|------|----"
+	out[0] = "Namespace|Name|Status|Type|Version"
+	out[1] = "---------|----|------|----|-------"
 	for i, items := range cvols.Items {
 		if len(items.Status.Reason) == 0 {
 			items.Status.Reason = volumeStatusOK
 		}
-		out[i+2] = fmt.Sprintf("%s|%s|%s|%s", items.ObjectMeta.Namespace,
+		out[i+2] = fmt.Sprintf("%s|%s|%s|%s|%s", items.ObjectMeta.Namespace,
 			items.ObjectMeta.Name,
-			items.Status.Reason, items.Spec.CasType)
+			items.Status.Reason, items.Spec.CasType, items.ObjectMeta.ResourceVersion)
```

Comment:
@RajatVaryani 
We need not get the resource version of the object.
We need to get the openebs version for the volume.
cc @ashishranjan738

Reply:
@ashishranjan738 Can I get confirmation on what @sonasingh46 is telling?

Reply:
yes @RajatVaryani we want openebs volume version, not the ResourceVersion. I think it not available in the response from apiserver. You also need to make changes to the backend for getting this info. Thanks for working on this issue